From f0714d41c06742463e0480849b2a341c39968b8f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Melissa=20Weber=20Mendon=C3=A7a?= <melissawm@gmail.com>
Date: Fri, 14 Apr 2023 23:55:55 -0300
Subject: [PATCH] Update URL of version switcher (#139)

# Description
Update URL of version switcher to use
https://napari.org/dev/_static/version_switcher.json

This brings us closer to the goal of napari.github.io containing
only built artifacts, not source materials.

## Type of change
<!-- Please delete options that are not relevant. -->
- [ ] Fixes or improves existing content
- [ ] Adds new content page(s)
- [x] Fixes or improves workflow, documentation build or deployment

# References
This is a follow-up on #109.

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added [alt text](https://webaim.org/techniques/alttext/) to
new images included in this PR
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 2808d293..72fae9df 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -83,7 +83,7 @@
 html_theme = 'napari'
 
 # Define the json_url for our version switcher.
-json_url = "https://napari.org/version_switcher.json"
+json_url = "https://napari.org/dev/_static/version_switcher.json"
 
 version_match = "latest" if version == "dev" else release

